pkg/testutil/testutil.go

```diff
@@ -487,7 +487,7 @@ func (c *Cmd) OutLines() []string {
 	c.Base.T.Helper()
 	out := c.Out()
 	// FIXME: improve memory efficiency
-	return strings.Split(out, "\n")
+	return strings.Split(strings.TrimSpace(out), "\n")
```

Comment:
This change is because otherwise it produces an empty string, which is useless and when used in follow up cmds (e.g. `rmi`) causes errors like this:

```
filters: parse error: [name==]: expected value or quoted: invalid argument: invalid argument"
```